src/yarp.c

```diff
@@ -8203,22 +8206,25 @@ parse_target(yp_parser_t *parser, yp_node_t *target) {
                     return target;
                 }
 
-                // The method name needs to change. If we previously had foo, we now
-                // need foo=. In this case we'll allocate a new owned string, copy
-                // the previous method name in, and append an =.
-                size_t length = yp_string_length(&call->name);
+                if (*call->message_loc.start == '_' || parser->encoding.alnum_char(call->message_loc.start, call->message_loc.end - call->message_loc.start)) {
+                    // The method name needs to change. If we previously had
+                    // foo, we now need foo=. In this case we'll allocate a new
+                    // owned string, copy the previous method name in, and
+                    // append an =.
+                    size_t length = yp_string_length(&call->name);
 
-                uint8_t *name = calloc(length + 1, sizeof(uint8_t));
-                if (name == NULL) return NULL;
+                    uint8_t *name = calloc(length + 1, sizeof(uint8_t));
+                    if (name == NULL) return NULL;
```

Comment:
In what case would `name == NULL`?

Reply:
Any memory allocation functions can return `NULL` (malloc, calloc, realloc). This is providing a slightly nicer experience in the case that we're out of memory.